### ISSUE TITLE
Tabs replaced by Spaces

### ISSUE THREAD

**Issue by DanielMurphy22** [mantidproject/mantid#27557]
**Original reporter** [James Lord]/[ISIS]

Workbench replaces tabs with four spaces whenever read through Python (ie if you print a string with a tab in it). It is NOT the case that pasting or opening in WB, displays tabs as spaces in the Script Editor. Only when Python reads it.

Run this in Workbench 

```
mystring= "hello	world" # a real tab
for i in mystring:
    print(ord(i))
print (repr(mystring))
```

Please see original [Mantid Forum Post](https://forum.mantidproject.org/t/workbench-replacing-tabs-with-spaces-in-string-literals/602)

**Comment by DanielMurphy22** [comment]
Another example of this:
```
spaces="        " # some spaces
tabs="			" # some tabs (paste in)
print ("spaces", list(ord(x) for x in spaces))
print ("tabs", list(ord(x) for x in tabs))
```
Notice how 8 spaces are recognised as 8 spaces but 3 tabs are recognised as 12 spaces.

From the [Mantid Forum](https://forum.mantidproject.org/t/workbench-python-bugs-python-3-release/636)

**Comment by NickDraper** [comment]
fixed by #27986
